src/main/java/nl/novi/serviceconnect/services/ServiceService.java

```diff
@@ -13,58 +13,84 @@
 
 @Service
 public class ServiceService implements IServiceService {
-    private final ServiceRepository repo;
+    private final ServiceRepository repository;
 
     public ServiceService(ServiceRepository repo) {
-        this.repo = repo;
+        this.repository = repo;
     }
 
     @Override
-    public ServiceInputDto CreateService(ServiceInputDto serviceInputDto) {
-        repo.save(Mapper.fromDtoToService(serviceInputDto));
+    public ServiceInputDto createService(ServiceInputDto serviceInputDto) {
+        repository.save(Mapper.fromDtoToService(serviceInputDto));
         return serviceInputDto;
     }
 
     @Override
-    public List<ServiceOutputDto> GetAllService() {
-        List<nl.novi.serviceconnect.models.Service> serviceList = repo.findAll();
+    public List<ServiceOutputDto> getAllService() {
+        List<nl.novi.serviceconnect.models.Service> serviceList = repository.findAll();
         List<ServiceOutputDto> serviceOutputDtos = new ArrayList<>();
 
+        if (serviceList.isEmpty()) {
+            throw new RecordNotFoundException("No services found");
+        }
+
         for(nl.novi.serviceconnect.models.Service service : serviceList) {
             serviceOutputDtos.add(Mapper.fromServiceToDto(service));
         }
+
         return serviceOutputDtos;
     }
 
     @Override
-    public ServiceOutputDto GetServiceById(Long id) {
+    public ServiceOutputDto getServiceById(Long id) {
+        Optional<nl.novi.serviceconnect.models.Service> serviceOptional = repository.findById(id);
```

Comment:
Gebruik het type van een variabele niet in de naamgeving. Hongaarse notatie is achterhaald. De naam van een variabele moet slaan op waar het voor is.